src/main.rs

```diff
@@ -49,18 +50,25 @@ enum Cmd {
         /// - `bootleby.bin` gives the Bootleby code as a raw image.
         bundle: PathBuf,
         /// Requires that a subset of keys are enabled (and only that subset).
-        /// This flag takes a bitmask (in the base of your choice as long as
-        /// it's not octal) where bit 0 maps to RoTK 0, 1 maps to 1, and so
-        /// forth. The install will proceed only if the bundle's CFPA is
-        /// configured such that keys corresponding to 1 bits are enabled and
-        /// keys corresponding to 0 bits are not enabled yet.
-        #[clap(long, value_parser = parse_int::parse::<u8>)]
-        require_key_enable_shape: Option<u8>,
+        /// This flag takes a 4 character string where 'E' represents
+        /// enabled, 'R' revoked, and 'I' invalid. Keys are specified
+        /// in order from 3 to 0. A string of EERR means keys 0 and 1
+        /// are revoked and keys 2 and 3 are enabled.
+        #[clap(long, value_parser = KeyStateTable::from_str)]
```

Comment:
I think you can now remove `value_parser = KeyStateTable::from_str`; `clap` is smart enough to use it automatically.

(same applies in two places below).